src/MsgPopup.qml

```diff
@@ -137,6 +139,15 @@ Popup {
                 font.family: roboto.name
                 visible: msgpopup.quitButton
             }
+            
+            ImButton {
+                text: qsTr("QUICK INSTALL GUIDE")
+                onClicked: {
+                    imageWriter.openUrl("https://docs.unraid.net/unraid-os/getting-started/quick-install-guide/")
```

Comment:
Please use a "go" link here - see the comment linked for more information!

https://github.com/unraid/usb-creator-next/issues/23#issuecomment-2263892277